packages/core/src/fields/types/timestamp/tests/test-fixtures.ts

```diff
@@ -33,77 +32,3 @@ export const storedValues = () => [
   { name: 'person6', lastOnline: null },
   { name: 'person7', lastOnline: null },
 ];
-
-export const filterTests = (withKeystone: (args: any) => any) => {
```

Comment:
These tests are not running at present, and we aren't going to change that in this pull request.